api/src/paths/xlsx/process.ts

```diff
@@ -111,10 +112,17 @@ export function processFile(): RequestHandler {
       await connection.open();
 
       const validationService = new ValidationService(connection);
-      // process the raw template data
-      await validationService.processXLSXFile(submissionId, surveyId);
-      // process the resulting transformed dwc data
-      await validationService.processDWCFile(submissionId);
+
+      try {
+        // process the raw template data
+        await validationService.processXLSXFile(submissionId, surveyId);
+        // process the resulting transformed dwc data
+        await validationService.processDWCFile(submissionId);
+      } catch (error: any) {
+        if (!(error instanceof SubmissionError)) {
```

Comment:
Might be good to add a comment here briefly explaining why we only throw non `SubmissionError` errors.